scripts/windows/install_erlang.bat

```diff
@@ -0,0 +1,73 @@
+@echo off
+:: Pass -v as first argument to enable debug echo
+:: Script to download and install Erlang/OTP from %OTP_URL% which defaults to a fast mirror
+:: Erlang/OTP versions could be switched by providing different <otp_version> for the same <install_path>
+:: Output is suitable for piping scripts
+:: ENV vars:
+::      WIN_OTP_PATH
+::      OTP_VERSION
+::      ERTS_VERSION
+::      OTP_URL
+:: Usage: install_erlang [/?] | <otp_version> [<install_path>]
+
+SETLOCAL ENABLEEXTENSIONS
+@IF "%~1"=="-v" shift /1 & echo on & echo
+
+IF NOT "%~1"=="/?" IF NOT "%~1"=="--help" IF NOT "%~1"=="" GOTO:START
+@echo Download and install Erlang/OTP from erlang-solutions.com
+@echo Usage: %0 [/?] ^| ^<otp_version^> [^<install_path^>]
+@echo.
+@echo Notes:
+@echo     Default ^<install_path^> is "C:\tools\erl" or WIN_OTP_PATH env var
+@echo     Define OTP_URL env var to use different download url
+@echo     Any existing installs in ^<install_path^> will be uninstalled first
+@echo     Env vars required by build scripts will be dumped
+@IF NOT "%WIN_OTP_PATH%"=="" echo     WIN_OTP_PATH=%WIN_OTP_PATH%
+exit /b 2
+
+:START
+IF NOT "%~2"=="" SET "WIN_OTP_PATH=%~2"
+IF "%WIN_OTP_PATH%"=="" SET "WIN_OTP_PATH=C:\tools\erl%~1"
+
+:: Check if we have old install
+call:get_version OTP_VERSION ERTS_VERSION
+
+:: Installed version is different
+IF NOT "%OTP_VERSION%"=="%~1" GOTO:INSTALL
+IF EXIST "%WIN_OTP_PATH%\bin\" @call:log Erlang/OTP %OTP_VERSION% && GOTO:DONE
+:INSTALL
+@call:log Install Erlang/OTP %~1 into %WIN_OTP_PATH%
+IF "%OTP_URL%"=="" SET "OTP_URL=https://packages.erlang-solutions.com/erlang/erlang/esl-erlang/FLAVOUR_1_general/esl-erlang_%~1~windows_amd64.exe"
+@call:log Download from %OTP_URL%
+PowerShell (New-Object System.Net.WebClient).DownloadFile('%OTP_URL%', '%TMP%\erl.exe')
+@IF NOT "%ERRORLEVEL%"=="0" (call:log Download failed && exit /b %ERRORLEVEL%)
+
+:: Uninstall the old version
```

Comment:
Just in case we have old version of Erlang in the same directory we have to uninstall it first. If we install in a separate folder <install_path> we can have multiple versions simultaneously. Then switching the active can be done via ENV